src/app/accounts/login.component.html

```diff
@@ -36,6 +36,7 @@
                         <label class="form-check-label" for="rememberEmail">{{'rememberEmail' | i18n}}</label>
                     </div>
                     <hr>
+                    <div [hidden]="!showCaptcha()"><iframe id="hcaptcha_iframe" height=80></iframe></div>
```

Comment:
It would be awesome if this was centered similar to the buttons below it (same as the other forms, should be aligned more with the buttons below it since it's a blocky looking element located just above the buttons).

Reply:
Better?
![image](https://user-images.githubusercontent.com/18214891/126654794-1814fe3a-6d9b-4d19-b1e3-3bdd01ce47ec.png)

Reply:
I don't really like either. I wanted it to be the full width but I don't want to mess with the internal styling of the element.

I went with aligned to the other inputs but I can see your logic too.  Maybe we should move above the <hr> and left-align?

Reply:
I agree that full width would be ideal; but if that is not an option due to lift, I prefer left aligned.

Reply:
Sorry @MGibson1 I missed the note to move the captcha above the line; I agree with this idea as well. It helps organize all the fields the user needs to address in one area with the action buttons below. 
<img width="334" alt="Screen Shot 2021-07-22 at 8 49 28 AM" src="https://user-images.githubusercontent.com/43477473/126669124-8b7f97bf-ac9f-4569-860f-ce95fa83fda3.png">